cdp_scrapers/legistar_utils.py

```diff
@@ -382,7 +383,7 @@ def get_events(
         get_legistar_events_for_timespan
         """
         if begin is None:
-            begin = datetime.utcnow() - timedelta(days=2)
+            begin = datetime.utcnow() - timedelta(days=40)
```

Comment:
Please leave this at `days=2`

Reply:
Forgot to revert back this change. will revert it back.